res/values/strings.xml

```diff
@@ -245,18 +245,18 @@
     <string name="ConversationFragment_quoted_message_not_found">Original message not found</string>
     <string name="reply_privately">Reply privately</string>
 
-    <string name="mute_for_one_hour">Mute for 1 hour</string>
-    <string name="mute_for_two_hours">Mute for 2 hours</string>
-    <string name="mute_for_one_day">Mute for 1 day</string>
-    <string name="mute_for_seven_days">Mute for 7 days</string>
+    <string name="mute_for_one_hour">Mute one hour</string>
+    <string name="mute_for_two_hours">Mute two hours</string>
+    <string name="mute_for_one_day">Mute one day</string>
+    <string name="mute_for_seven_days">Mute seven days</string>
```

Comment:
not sure, if these written numbers here and at similar places an improvement, although maybe more correct.

Reply:
Just looks more natural. Technically it is probably a list and so on, but in translation you avoid problems with languages that write out all numbers 1-10.